roles/pulp-api/templates/pulp-api.deployment.yaml.j2

```diff
@@ -49,6 +49,9 @@ spec:
       tolerations:
         {{ tolerations | indent(width=8) }}
 {% endif %}
+      ulimits:
```

Comment:
@fao89 @git-hyagi can someone guide me a tell me if i was supposed to rather make this change in podman-compose.yml?

Reply:
I don't think it is needed

Reply:
hum... not sure about how to do it, but it seems like it is not possible through a pod spec yet:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#podspec-v1-core (there isn't an `ulimits` field)

and checking kubernetes repo, this seems to be a long discussion:
https://github.com/kubernetes/kubernetes/issues/92778
https://github.com/kubernetes/kubernetes/issues/3595

Reply:
well, that's a bummer :/ The discussion has been opened for 5+ years.

Will putting this ulimit directly into compose file will help?

Reply:
compose is more for people to read than use it

Reply:
so, i am closing then this PR, apparently.
However I am still not sure how we'd handle situation when we run out of file descriptors?

Reply:
> so, i am closing then this PR, apparently. However I am still not sure how we'd handle situation when we run out of file descriptors?

Let's cross this bridge and then point to https://issues.redhat.com/browse/RFE-2000